apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.js

```diff
@@ -41,21 +42,9 @@ function checkSendingPossibility (newsItem) {
     return true
 }
 
-/**
- * @param {NewsItem} newsItem
- * @returns {Promise<void>}
- */
-async function sendNotifications (newsItem) {
-
-    if (!checkSendingPossibility(newsItem)) {
-        return
-    }
```

Comment:
Please put this checking back. We need to check actual news item data before sending a message. For example, the news item may be canceled (removed).

Reply:
Brought back